dist/browser/QuillDeltaToHtmlConverter.bundle.js

```diff
@@ -1208,10 +1301,10 @@ exports.tokenizeWithNewLines = tokenizeWithNewLines;
 },{}],15:[function(require,module,exports){
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
-function sanitize(str) {
+function sanitize(str, urlWhiteListExtensions) {
     var val = str;
     val = val.replace(/^\s*/gm, '');
-    var whiteList = /^\s*((|https?|s?ftp|file|blob|mailto|tel):|#|\/|data:image\/)/;
+    var whiteList = new RegExp("^s*((|https?|s?ftp|file|blob|mailto|tel):" + (urlWhiteListExtensions ? '|' + urlWhiteListExtensions.join('|') : '') + "|#|/|data:image/)");
```

Comment:
begins with `s` instead of begins with whitespace `\s`?

Reply:
this is the built file my plain ts file does not have this change: https://github.com/nozer/quill-delta-to-html/pull/51/files#diff-f6267fface5ff43131ebc65c1d46bb65
would like a maintainer to chime in on why the build modified the regex like this

Reply:
```javascript
let whiteList = new RegExp(`^\s*((|https?|s?ftp|file|blob|mailto|tel):${urlWhiteListExtensions ? '|' + urlWhiteListExtensions.join('|') : ''}|#|\/|data:image\/)`)
```

You would need to escape `\` with multiple `\`s there.